### ISSUE TITLE
Only latest annotation with BrushLabel is saved in Completion

### ISSUE THREAD

**Issue by a99user** [heartexlabs/label-studio#337]
**Describe the bug**
I open already submitted Completion and update it with Brush and save changes, after opening it again only latest mask is visible (as well as content of completion .json is rewritten i.e. size is significantly smaller if new mask is smaller).

**To Reproduce**

1. Use following config:

```
<View>
    <BrushLabels name="tag" toName="img">
        <Label value="Label" background="blue" />
    </BrushLabels>
    <Image name="img" value="$image" zoom="true" zoomControl="true" />
</View>
```

2. Create annotation task and open it for labeling.
3. Create big mask with brush and submit.
4. JSON file for the completion appeared on disk in ./label-studio/my_project/completions and its size is 3 Mb.
5. Open completion again.
6. Either create new brush mask or click on existing and create additional mask segment (smaller size) and press Update button.
7. Size of the same JSON file changed to 246K.
8. Open task for labeling again.
9. Only second mask is displayed.

Please note that the same behavior is observed if new Brush region created instead of extending the existing one.

**Expected behavior**
The initial segmentation mask is extended with added segment. Previously created mask is not disappeared.

**Screenshots**
Step 3:
![1](https://user-images.githubusercontent.com/59999602/86168549-0a56d880-bb21-11ea-8589-26b2cade377f.jpg)
Step 6:
![2](https://user-images.githubusercontent.com/59999602/86170291-91a54b80-bb23-11ea-9f5d-8d08681d304e.jpg)
Step 9:
![3](https://user-images.githubusercontent.com/59999602/86170328-a4b81b80-bb23-11ea-8256-0d2303243683.jpg)

**Environment (please complete the following information):**

- OS: Ubuntu 18.04
- Browser: Chrome
- Version: 83
- docker-compose version: 1.26.0
- docker version: 19.03.12

**Comment by makseq** [comment]
@a99user We've just released label-studio 0.7.4rc2 with a lot of fixes regarding to brushes, including this issue. If it's still actual, please check it. Also we've added brush converters on export page to save the results as numpy 2d arrays and PNG.

```
pip uninstall label-studio
pip install label-studio==0.7.4rc2
```

**Comment by makseq** [comment]
Fixed in https://github.com/heartexlabs/label-studio/releases/tag/v0.7.4
